hivemq-edge/src/test/java/com/hivemq/adapter/AdapterConfigFormatTest.java

```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright 2019-present HiveMQ GmbH
```

Comment:
@DC2-DanielKrueger I think this is actually 2023. (Not that it makes a huge difference, but 4 years more IP coverage would be nice ;) )